### DISCUSSION TITLE
Can someone please send me a slack invite?

### DISCUSSION THREAD

**Discussion by groobyming** [milvus-io/milvus#26083] [Q&A and General discussion]
I can't join the slack community using the official link
<img width="803" alt="image" src="https://github.com/milvus-io/milvus/assets/34335395/7d7e7f66-742c-4ede-9936-43a36ebb1009">

**Comment by JackLCL** [comment]
Can you tell us which link you use? I try the link on milvus.io. It's fine. Maybe your network is not healthy.

**Comment by groobyming** [comment]
@JackLCL 
https://milvus.io/slack

**Comment by JackLCL** [comment]
![image](https://github.com/milvus-io/milvus/assets/53512883/7b6642d1-6e9a-402d-9343-8deb6ca42c7e)
It's OK for me.

**Reply by caozhuozi** [reply]
@JackLCL  I met the same issue with @groobyming. Could you please help check?

Besides, I already sent about 3   invite requests before, but got no response. Could you pleask help check  the invite request from wxdahk@gmail.com ?

**Comment by yhmo** [comment]
@groobyming 
Provide your email address, I will send an invite email to you from slack.
